apps/emqx_management/etc/emqx_management.conf

```diff
@@ -17,6 +17,7 @@ emqx_management:{
             send_timeout_close: on
             inet6: false
             ipv6_v6only: false
+            ip: "0.0.0.0"
```

Comment:
listener:0.0.0.0:8081

Reply:
Ip port merger?

Reply:
Some users already have port configured, this way it won't be necessary to migrate any configuration. Also for ipv6 it will look  a bit busy `[::1]:8081`

Reply:
https://github.com/emqx/hocon/blob/master/sample-schemas/emqx_management_schema.erl#L7